src/Controller/BaseballGame.js

```diff
@@ -0,0 +1,62 @@
+const { Console } = require('@woowacourse/mission-utils');
+const Utils = require('../Util/Utils');
+const InputView = require('../View/InputView');
+const OutputView = require('../View/OutputView');
+const Validator = require('../Model/Validator');
+const { OPTION, NUMBER } = require('../Util/Constants');
+const Computer = require('../Model/Computer');
```

Comment:
선언부를 같은 부류끼리 묶어두면 나중에 볼륨이 커졌을 경우 찾기에 좋을 것 같습니다 :)

```suggestion
/** 유틸 관련 임포트 */
const Utils = require('../Util/Utils');
const { Console } = require('@woowacourse/mission-utils');

/** View 관련 임포트 */
const InputView = require('../View/InputView');
const OutputView = require('../View/OutputView');

/** 모델 관련 임포트 */
const Validator = require('../Model/Validator');
const Computer = require('../Model/Computer');

/** 상수 임포트 */
const { OPTION, NUMBER } = require('../Util/Constants');
```